release-notes.md

```diff
@@ -4,6 +4,7 @@
 
 - qemu:
   * Updated to QEMU 6.2 release.
+  * Added LSM303DLHC magnetometer support.
```

Comment:
This PR will actually pull in a number of optional I2C sensors if `I2C_DEVICES` is enabled during QEMU builds, although LSM303DLHC is the main use case since Zephyr has support for that device out of the box, and was what I was hoping to enable in CI for testing I2C.

Reply:
Specifically:

RTC:

- config DS1338
- config M41T80

Sensors:

- config TMP105
- config TMP421
- config DPS310
- config EMC141X
- config ADM1272
- config MAX34451
- config LSM303DLHC_MAG

Reply:
> This PR will actually pull in a number of optional I2C sensors if `I2C_DEVICES` is enabled during QEMU builds, although LSM303DLHC is the main use case since Zephyr has support for that device out of the box, and was what I was hoping to enable in CI for testing I2C.

ok, I will try to update the commit message and the release notes to reflect that.